content/chainguard/chainguard-enforce/vulnerabilities/vuln-reports-and-attestations.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Vulnerability reports and Attestations in Chainguard Enforce"
+linktitle: "Vulnerability reports and Attestations"
+aliases:
+- /chainguard/chainguard-enforce/chainguard-enforce-kubernetes/vuln-reports-and-attestations/
+- /chainguard/chainguard-enforce/concepts/vuln-reports-and-attestations/
+type: "article"
+description: "An overview of how Chainguard Enforce handles attestations of vulnerability reports"
+date: 2023-03-19T15:56:52-07:00
+lastmod: 2023-03-19T15:56:52-07:00
+draft: false
+tags: ["ENFORCE", "COSIGN", "VULNERABILITIES", "CONCEPTUAL"]
+images: []
+menu:
+  docs:
+    parent: "concepts"
+weight: 030
+toc: true
+---
+
+In order to fully leverage the capabilities offered by Chainguard Enforce, it's important to establish a process for generating quality vulnerability reports.
+A vulnerability report lists all the discovered vulnerabilities contained in your software. It uses a [SBOM](/open-source/sbom/) to search for vulnerabilities matching the used packages in your software.
```

Comment:
```suggestion
A vulnerability report lists all the discovered vulnerabilities contained in your software. Vulnerability scanning tools can use [SBOMs](/open-source/sbom/) to search for vulnerabilities that match the packages used to build your application.
```

Slightly reworded this to include scanner tools, since Grype is going to be introduced later on.